src/main/java/edu/iis/powp/events/predefine/SelectTestFigureOptionListener.java

```diff
@@ -13,6 +13,14 @@
     @Override
     public void actionPerformed(ActionEvent e)
     {
-        FiguresJoe.figureScript1(Application.getComponent(DriverManager.class).getCurrentPlotter());
+    	String command = e.getActionCommand();
+    	switch(command) {
```

Comment:
Separate class for Joe2 test would be better. 
It is bad to rely on a name of the menu element.